### PR TITLE
HAR/TU WRF-Mercator

### PR DESCRIPTION
**Pull request by benatouba**
Add support for WRF-Mercator projection in TU style products

### CODE REVIEW

**Comment by fmaussion** [comment]
Could you make a clean PR without the previous commit?

File: salem/sio.py [hunk 1]
```diff
@@ -126,22 +126,25 @@ def read_shapefile_to_grid(fpath, grid):
...
         # HAR and other TU Berlin files
-        dx = ds.GRID_DX
-        dy = ds.GRID_DY
-        pargs['lat_1'] = ds.PROJ_STANDARD_PAR1
-        pargs['lat_2'] = ds.PROJ_STANDARD_PAR2
-        pargs['lat_0'] = ds.PROJ_CENTRAL_LAT
-        pargs['lon_0'] = ds.PROJ_CENTRAL_LON
-        pargs['center_lon'] = ds.PROJ_CENTRAL_LON
+        dx = ds.GRID_DX or ds.DX
+        dy = ds.GRID_DY or ds.DY
```
Comment (fmaussion) [inline salem/sio.py:133]: This doesnt look like a good idea because if GRID_DX is not available it will raise an error, not go to ds.DX

**Comment by benatouba** [comment]
Did the rebase now. Adding a fix for your remark next

**Comment by benatouba** [comment]
As for testing, i can upload a small test file to the sample data repository

**Comment by fmaussion** [comment]
Nah thanks, it's ok like this -  I'm still puzzled as to why the tests fail, let's check on master

**Pull request merged**